src/main/java/net/openhft/chronicle/core/Maths.java

```diff
@@ -16,6 +16,7 @@
 
 package net.openhft.chronicle.core;
 
+import net.openhft.chronicle.core.annotation.Java9;
```

Comment:
This import doesn't seem to be used anywhere - can it be removed?

Reply:
You are right. Fixed.